src/home/components/home-banner/HomeBanner.tsx

```diff
@@ -0,0 +1,60 @@
+import { useState } from 'react';
+import { useNavigate } from 'react-router';
+import { getConfig } from '@edx/frontend-platform';
+import { useIntl } from '@edx/frontend-platform/i18n';
+import { Form, useToggle, SearchField } from '@openedx/paragon';
+
+import { ROUTES } from '@src/routes';
+import HomeOverlayHtmlSlot from '@src/plugin-slots/HomeOverlayHtmlSlot/HomeOverlayHtmlSlot';
+import HomePromoVideoButtonSlot from '@src/plugin-slots/HomePromoVideoButtonSlot/HomePromoVideoButtonSlot';
+import HomePromoVideoModalSlot from '@src/plugin-slots/HomePromoVideoModalSlot/HomePromoVideoModalSlot';
+
+import messages from './messages';
+
+const HomeBanner = () => {
+  const intl = useIntl();
+  const navigate = useNavigate();
+  const [searchValue, setSearchValue] = useState('');
+  const [isOpen, open, close] = useToggle(false);
+
+  const handleSearch = () => navigate(`${ROUTES.COURSES}?search_query=${searchValue}`);
+
+  const searchField = getConfig().ENABLE_COURSE_DISCOVERY && (
+    <Form.Group className="w-100 mb-0 mt-4.5">
+      <SearchField
+        placeholder={intl.formatMessage(messages.searchPlaceholder)}
+        value={searchValue}
+        onChange={(value: string) => setSearchValue(value)}
+        onKeyDown={(e: React.KeyboardEvent<HTMLInputElement>) => {
+          if (e.key === 'Enter') {
+            e.preventDefault();
+            handleSearch();
+          }
+        }}
+        onSubmit={() => {}}
+      />
+    </Form.Group>
+  );
+
+  return (
+    <section
+      className="home-banner d-flex justify-content-center align-items-center position-relative overflow-hidden"
+      data-testid="home-banner"
+    >
+      <div className="outer-wrapper d-flex justify-content-center align-items-center flex-column mx-auto mt-0 p-4">
```

Comment:
It's not clear to me what `mx-auto` and `mt-0` are doing here. `margin-top` seems to already be `0` without `mt-0`, and removing `mx-auto` doesn't appear to change anything visually.